mailet/api/src/main/java/org/apache/mailet/Mail.java

```diff
@@ -243,6 +243,41 @@ default boolean hasSender() {
      */
     Optional<Attribute> getAttribute(AttributeName name);
 
+    /**
+     * Returns the raw attribute value corresponding to an attribute name.
+     *
+     * A cast will be attempted to match the type expected by the user.
+     *
+     * Returns an empty optional upon missing attribute or type mismatch.
+     *
+     * @since Mailet API v3.2
+     */
+    default <T> Optional<T> getRawAttributeValue(AttributeName name, Class<T> type) {
```

Comment:
For me, this part have to be removed, that's not part of the `Mail`'s responsibility.

Reply:
This method looks like a deprecated one
```java
Serializable getAttribute(String name);
```
It takes the key, and return the value from a mail. With me, regards to the purpose of both methods. They are the same. (give the key, get the value)